django_freeradius/settings.py

```diff
@@ -2,3 +2,17 @@
 
 EDITABLE_ACCOUNTING = getattr(settings, 'DJANGO_FREERADIUS_EDITABLE_ACCOUNTING', False)
 EDITABLE_POSTAUTH = getattr(settings, 'DJANGO_FREERADIUS_EDITABLE_POSTAUTH', False)
+
+DEFAULT_SECRET_FORMAT = 'NT-Password'
+
+DISABLED_SECRET_FORMAT = ['Cleartext-Password',
+                          'LM-Password',
+                          'MD5-Password',
+                          'SMD5-Password',
+                          'SSHA-Password',
+                          'Crypt-Password']
```

Comment:
I should not agree with this because this implies that another definition in global settings.py should be done. Isn't so?

Reply:
@peppelinux no definition in global `settings.py` is required, because the value we set there will become the default, but we need to add that construct otherwise users won't be able to override the default value by editing their own global `settings.py`, right now it basically the same as if you hardcoded the values directly in the main code